alerts/cluster-monitoring-operator/NodeClockNotSynchronising.md

```diff
@@ -0,0 +1,50 @@
+# NodeClockNotSynchronising
+
+## Meaning
+
+This alert signifies issues with the NTP server on a node (mentioned in the alert).
+When the affected node is a master, this specifically becomes an issue
+as when the certs are rotated for the apiserver on that node, they fail
+validation due to the invalid time on that master.
+
+
+## Impact
+This alert is critical.
```

Comment:
The rule says otherwise - https://github.com/openshift/cluster-monitoring-operator/blob/320ea1b465d8aa398ae1aefa05e67dc048fa33dc/assets/node-exporter/prometheus-rule.yaml#L216

Reply:
Hello, so we're wanting to change that rule from warning to critical. i was under the impression that i need an approved runbook before having a patch for CMO. let me make a patch as well and link it here!

Reply:
Thank you foe letting me know. i'll check up with them

Reply:
Thanks for chiming in @simonpasquier 🙇